src/include/function/gds/gds_function_collection.h

```diff
@@ -7,31 +7,31 @@ namespace kuzu {
 namespace function {
 
 struct WeaklyConnectedComponentsFunction {
-    static constexpr const char* name = "WEAKLY_CONNECTED_COMPONENT";
+    static constexpr const char* name = "GDS_WCC";
```

Comment:
revert all naming changes in this PR. We should add alias naming instead of renaming.